src/pages/settings/invoice-design/pages/template-designs/TemplateDesigns.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { DataTable } from '$app/components/DataTable';
+import { EntityStatus } from '$app/components/EntityStatus';
+import { Inline } from '$app/components/Inline';
+
+export default function TemplateDesigns() {
+    return (
+        <DataTable
+            endpoint="/api/v1/designs?template=true"
+            columns={[
+                {
+                    id: 'name',
+                    label: 'Name',
```

Comment:
Does this work? We use labels more like placeholder keys. Like, $model[$label]. Not sure if an uppercased one works?